test/sslapitest.c

```diff
@@ -656,6 +657,178 @@ static int execute_test_large_message(const SSL_METHOD *smeth,
     return testresult;
 }
 
+#ifndef OPENSSL_NO_KTLS
```

Comment:
We'll get failures from these tests if ktls was enabled but was not available in this kernel version.

Reply:
Should be fixed, see previous comment about runtime check.

Reply:
Yes this works now.